sys-auth/sssd/files/sssd-1.13.1-fix-init.patch

```diff
@@ -0,0 +1,12 @@
+--- sssd-1.13.1-org/src/sysv/gentoo/sssd.in	2015-09-30 12:04:16.000000000 +0200
++++ sssd-1.13.1/src/sysv/gentoo/sssd.in	2017-08-06 17:31:30.692971705 +0200
```

Comment:
please try to git imitate patches, i.e. make the header:
```
--- a/src/sysv/gentoo/sssd.in
+++ b/src/sysv/gentoo/sssd.in
```

1. Least number of bytes and still -p1
2. Version agnostic
3. Dates add no value